app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizard.js

```diff
@@ -1,33 +1,151 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { noop } from 'patternfly-react';
-import ModalWizard from '../../components/ModalWizard';
+import {
+  bindMethods,
+  noop,
+  Button,
+  Icon,
+  Modal,
+  Wizard
+} from 'patternfly-react';
+import { createMigrationPlans } from './helpers';
 import PlanWizardBody from './PlanWizardBody';
 
-const PlanWizard = ({
-  hidePlanWizard,
-  hidePlanWizardAction,
-  planWizardExitedAction
-}) => (
-  <ModalWizard.StateProvider numSteps={3}>
-    <ModalWizard
-      showWizard={!hidePlanWizard}
-      onHide={hidePlanWizardAction}
-      onExited={planWizardExitedAction}
-      title={__('Migration Plan Wizard')}
-    >
-      <PlanWizardBody loaded />
-    </ModalWizard>
-  </ModalWizard.StateProvider>
-);
+const planWizardSteps = [
+  'planWizardGeneralStep',
+  'planWizardCSVStep',
+  'planWizardResultsStep'
+];
+
+class PlanWizard extends React.Component {
+  constructor() {
+    super();
+    this.state = { activeStepIndex: 0 };
+    bindMethods(this, ['prevStep', 'nextStep', 'goToStep']);
+  }
+
+  prevStep() {
+    const { activeStepIndex } = this.state;
+    this.setState({ activeStepIndex: Math.max(activeStepIndex - 1, 0) });
```

Comment:
Duplicating all this state stuff in both Wizards makes me sad... but to @serenamarie125's point on Slack yesterday, we have bigger fish to fry. I'll save the cleanup for when there is time.